plugins/modules/iam_user.py

```diff
@@ -27,6 +27,13 @@
     required: false
     type: str
     version_added: 2.2.0
+  password_reset_required:
+    description:
+      - Defines if the user is required to set a new password after login.
+    required: false
+    type: bool
+    default: false
+    version_added: 2.3.0
```

Comment:
Why `version_added` is `2.3.0`?

Reply:
Ah, good question.  
I think because I thought 2.2.0 is faster released. But because of https://github.com/ansible-collections/community.aws/pull/873 it must be 3.1.0.  
I'm going to increase it.